Examples/Resources/Extractors/SAT-features-competition2012_revised_without_SatELite_sparkle/sparkle_extractor_wrapper.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# -*- coding: UTF-8 -*-
+"""Template for users to create Feature Extractor wrappers."""
+
+import sys
+import os
+import time
+import random
+import argparse
+import subprocess
+from pathlib import Path
+
+global sparkle_special_string
+sparkle_special_string = r'__@@SPARKLE@@__'
+
+def get_last_level_directory_name(filepath):
+    return os.path.basename(filepath.rstrip('/'))
+
+def get_time_pid_random_string():
```

Comment:
don't use os.get_pid and for the randint use int(random.getrandbits(32)) instead

Reply:
Is there something I should use instead of os.get_pid?

Reply:
Nope